src/docfx/lib/JsonUtility.cs

```diff
@@ -286,20 +286,15 @@ public static (List<Error>, JToken) RemoveNulls(this JToken token)
             return (errors, token);
         }
 
-        public static bool TryGetValue<T>(this JObject obj, string key, out T value) where T : JToken
+        public static bool TryGetValue<T>(this JObject obj, string key, out T value)
         {
-            value = null;
-            if (obj == null || string.IsNullOrEmpty(key))
+            if (obj.TryGetValue(key, out var valueToken) && valueToken is JValue jvalue && jvalue.Value is T result)
```

Comment:
`valueToken is JValue` is too restrict? the method name can't tell me that i can't use this method is not a JValue